src/setupProxy.js

```diff
@@ -0,0 +1,11 @@
+const { createProxyMiddleware } = require('http-proxy-middleware');
+
+module.exports = (app) => {
+  app.use(
+    '/api',
+    createProxyMiddleware({
+      target: process.env.REACT_APP_PROXY_TARGET,
```

Comment:
Das hier ist der Proxy, den ich oben schon erwähnt habe. Das ist auch bei create-react-app automatisch dabei, siehe https://create-react-app.dev/docs/proxying-api-requests-in-development/

In dieser Zeile wird die Environment Variable verwendet, die wir in `.env` oder dann eben in `.env.local` verwenden.